pkg/config/default_linux.go

```diff
@@ -0,0 +1,19 @@
+package config
+
+import (
+	"syscall"
+)
+
+// isCgroup2UnifiedMode returns whether we are running in cgroup 2 cgroup2 mode.
```

Comment:
I know this is just a cut&paste, but is this comment sensible? And if not, would it be worth a recommit to fix it?